wordpress-mobile/WordPress-iOS
# Implement Stock Photos without WPMediaPicker

This PR reimplements Stocks Photos picker without the `WPMediaPicker` framework.

This is the final PR after which I can remove WPMediaPicker 😓 

## Changes

- Implement Stock photos picker using the new infra with `ExternalMediaPickerViewController`
- Change “Done” button to “Add”, including the Site Media picker
- Fix color of button “Show Selection”

## Testing

For every of the following flows:

- Add Image to Gutenberg
- Add Gallery to Gutenberg
- Add image to Aztec
- Add images to Site Media

Test the following:

- Verify that the new picker is displayed
- Verify that "Cancel" dismisses the screen and doesn't add any assets if the selection is made
- Verify that for multiple selection, the selection it oredered
- Verify that for multiple selection, a toolbar with a number of selected of items is displayed
- Verify that when you tap a toolbar button indicating the selection count the fullscreen gallery is displayed
- Verify that you can zoom in in the gallery
- Verify that when you tap "Done" the selected image are uploaded

## Regression Notes
1. Potential unintended areas of impact: Tenor (Aztec and Gutenberg post editor, Site Media screen)
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
